x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_classification.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiHealth, EuiSpacer, EuiSuperSelect, EuiTitle } from '@elastic/eui';
+import { findMaxMin, useDecisionPathData } from './use_classification_path_data';
+import { FeatureImportance, TopClasses } from '../../../../../common/types/feature_importance';
+import { DecisionPathChart } from './decision_path_chart';
+interface ClassificationDecisionPathProps {
+  predictedValue: string | undefined;
+  predictionFieldName?: string;
+  featureImportance: FeatureImportance[];
+  topClasses: TopClasses;
+}
+
+export const ClassificationDecisionPath: FC<ClassificationDecisionPathProps> = ({
+  featureImportance,
+  predictedValue,
+  topClasses,
+  predictionFieldName,
+}) => {
+  const [currentClass, setCurrentClass] = useState<string>(topClasses[0].class_name);
+  const { decisionPathData } = useDecisionPathData({
+    featureImportance,
+    predictedValue: currentClass,
+  });
+  const options = useMemo(
+    () =>
+      Array.isArray(topClasses) && typeof predictedValue === 'string'
+        ? topClasses.map((c) => ({
+            value: c.class_name,
+            inputDisplay:
+              c.class_name === predictedValue ? (
+                <EuiHealth color="success" style={{ lineHeight: 'inherit' }}>
+                  {c.class_name}
+                </EuiHealth>
+              ) : (
+                c.class_name
+              ),
+          }))
+        : undefined,
+    [topClasses, predictedValue]
+  );
+  const domain = useMemo(() => {
+    let maxDomain;
+    let minDomain;
+    // if decisionPathData has calculated cumulative path
+    if (
+      Array.isArray(decisionPathData) &&
+      decisionPathData.length > 0 &&
+      decisionPathData[0].length === 3
+    ) {
+      const { max, min } = findMaxMin(decisionPathData, (d: [string, number, number]) => d[2]);
+      const buffer = Math.abs(max - min) * 0.1;
+      maxDomain = max + buffer;
+      minDomain = min - buffer;
+    }
+    return { maxDomain, minDomain };
+  }, [decisionPathData]);
+
+  if (!decisionPathData) return <div />;
```

Comment:
Can you add a comment about reasons we could end up with returning an empty `div` here?

Reply:
Changed it to a EuiCallOut here https://github.com/elastic/kibana/pull/73561/commits/ef815d11a2bf334d703366daff3519b38b848e11